agent/src/main/java/com/cloud/agent/resource/consoleproxy/ConsoleProxyResource.java

```diff
@@ -83,16 +83,16 @@
 public class ConsoleProxyResource extends ServerResourceBase implements ServerResource {
     static final Logger s_logger = Logger.getLogger(ConsoleProxyResource.class);
 
-    private final Properties _properties = new Properties();
-    private Thread _consoleProxyMain = null;
+    private final Properties properties = new Properties();
+    private Thread consoleProxyMain = null;
 
-    long _proxyVmId;
-    int _proxyPort;
+    long proxyVmId;
+    int proxyPort;
 
-    String _localgw;
-    String _eth1ip;
-    String _eth1mask;
-    String _pubIp;
+    String localgw;
```

Comment:
@mlsorensen, as we are touching these declarations, what do you think about renaming some of them, like fixing the camelCase and describing them better?

For instance:

```suggestion
    String localGateway;
```

Reply:
Yes, I did get a few that seemed to have wrong capitalization. Hard for me to know what's right for all of them but I'll do a quick scan.